code/game/machinery/camera/camera.dm

```diff
@@ -43,22 +45,27 @@
 	assembly.update_icon()
 
 	GLOB.cameranet.cameras += src
-	GLOB.cameranet.addCamera(src)
+	part_of_camera_network = should_add_to_cameranet
+	if(part_of_camera_network)
+		GLOB.cameranet.addCamera(src)
 	if(isturf(loc))
 		LAZYADD(get_area(src).cameras, UID())
 	if(is_station_level(z) && prob(3) && !start_active)
-		toggle_cam(null, FALSE)
+		turn_off(null, FALSE)
 		wires.cut_all()
 
 /obj/machinery/camera/proc/set_area_motion(area/A)
 	area_motion = A
 
+/obj/machinery/camera/Moved(atom/OldLoc, Dir, Forced)
+	. = ..()
+	SEND_SIGNAL(src, COMSIG_CAMERA_MOVED, OldLoc)
```

Comment:
The parent sends this signal. Why send it again?

Reply:
It's a slightly different signal from `ATOM_MOVED`. `COMSIG_CAMERA_MOVED` is called whenever a camera has moved over the network. Since moving with a portable camera in hand won't trigger `ATOM_MOVED`. Or how borgs do it.
Probably not the best solution but I wanted to keep the behaviour of borg cameras largely the same.

Reply:
I thought this was atom moved. I’m a fool